Monitor_server.py

```diff
@@ -0,0 +1,58 @@
+import socket
+import sqlite3
+
+conn = sqlite3.connect('authentication_info.db')
+c = conn.cursor()
+c.execute('''DROP TABLE user_info''')
+c.execute("CREATE TABLE user_info (username text, password text)")
+c.execute("INSERT INTO user_info values('shashank','goud')")
+c.execute("INSERT INTO user_info values('ankit','bhandari')")
+c.execute("INSERT INTO user_info values('kausthub','sant')")
+c.execute("INSERT INTO user_info values('nikhil','chintapallee')")
+conn.commit()
+conn.close()
+               
+s = socket.socket()         
+host = socket.gethostname() 
+port = 13464               
+s.bind((host, port))        
+
+s.listen(5)                 
+while True:
+   c,addr = s.accept()     
+   print 'Got connection from', addr
+
+   data = c.recv(1024)
+   user_info = data.split(",")
+   username_input = user_info[0]
+   password_input = user_info[1]
+   conn_2 = sqlite3.connect('authentication_info.db')
+   c_2 = conn_2.cursor()
+   for row in c_2.execute("SELECT username from user_info where username = '%s'" % username_input):
```

Comment:
not required as the next query will return NULL if given username not in DB. 
So please remove this query and related statements.